src/axom/inlet/Function.hpp

```diff
@@ -48,12 +48,25 @@ namespace inlet
  * a 3D vector can be used in its place (with the third component being empty/zero)
  *******************************************************************************
  */
-enum class FunctionType
+enum class FunctionTag
 {
   Vec3D,
   Double
 };
 
+/*!
+ *******************************************************************************
+ * \brief The types used to describe function signatures in the input file
+ * 
+ * \note These are aliases intended to improve readability
+ *******************************************************************************
+ */
+struct FunctionType
+{
+  using Vec3D = axom::primal::Vector3D;
```

Comment:
Do you want to add Empty|None|whatever you want to call it in this PR or in a follow up?  String would also be helpful

Reply:
I'll save this for a follow up, I anticipate some nuance with the Void type